Python/other/longest_Substring.py

```diff
@@ -0,0 +1,41 @@
+def longestSubstringLength(string):
+    di = {}
+    result = 0
+    start = 0
+    counter = 0
+    for index, val in enumerate(string):
+        if val not in di:
+            di[val] = index
+            counter += 1
+        else:
+            if di[val] >= start:
+                start = di[val] + 1
+                counter = index - di[val] -1
+            counter += 1
+            di[val] = index
+        result = max(result, counter)
+    return result
+
+testCase = int(input('No. of test cases:'))
```

Comment:
Follow PEP8 Guidelines for variable naming & function naming conventions
https://www.python.org/dev/peps/pep-0008/#id45